azurerm/internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -214,6 +258,10 @@ func resourceArmLogAnalyticsWorkspaceRead(d *schema.ResourceData, meta interface
 		d.Set("daily_quota_gb", utils.Float(-1))
 	}
 
+	if err := d.Set("private_link_scoped_resource", flattenArmWorkspacePrivateLinkScopedResourceArray(resp.PrivateLinkScopedResources)); err != nil {
+		return fmt.Errorf("setting `private_link_scoped_resource`: %+v", err)
+	}
```

Comment:
(as above) this doesn't appear to be valuable to expose - so let's remove it

Reply:
once a workspace is associated to an azure monitor private link scope the private_link_scoped_resource property will have values, and then users can get the id of associated private link scope from it. Do you still think we need to remove it?

Reply:
It has been removed.